### PR TITLE
Added sign up screens and logic

### PR DESCRIPTION
**Pull request by bananapizzza**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -65,6 +66,8 @@ class MenuBody extends React.Component {
             return 'Calendar';
         } else if (pathname === '/my_account') {
             return (<MyAccount/>);
+        } else if(pathname === '/sign_up') {
```
Comment (xaveng) [inline src/App.js]: minor: indentation

File: src/actions/actions_signUp.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    const response = await fetch("http://localhost:5000/sign_up", {
+        method: "POST",
+        headers: {
+            'Content-type': 'application/json',
+        },
+        body: JSON.stringify(values)
+    });
+
+    const result = await response.json();
+    if(response.status >= 400) {
```
Comment (xaveng) [inline src/actions/actions_signUp.js]: Wouldn't it be better if we negate this if else statement by checking only HTTP 200 OK response?

File: src/components/SignUpComponent.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+import signUpInfoSubmit from "../actions/actions_signUp";
+
+const required = value => value ? undefined : 'Required';
+const passwordValidate = value => {
+    const regexForLength = new RegExp("(?=.{8,}).*");
+    const regexForLowerCase = new RegExp("(?=.*[a-z]).*");
+    const regexForUpperCase = new RegExp("(?=.*[A-Z]).*");
+    const regexForNumber = new RegExp("(?=.*[0-9]).*");
+
+    if(!regexForLength.test(value)){
```
Comment (xaveng) [inline src/components/SignUpComponent.js]: minor: indentation

File: src/actions/actions_signUp.js (hunk 2 above)
Comment (bananapizzza) [reply src/actions/actions_signUp.js]: Okay!

**Pull request merged**